lua/autorun/server/sv_cfc_contact_forms.lua

```diff
@@ -62,6 +62,7 @@ local function getE2Information()
             local e2Info = {}
             e2Info.name = expression2:GetGateName()
             e2Info.size = expression2:GetCode():len()
+            e2Info.cpuTime = expression2.context.timebench
```

Comment:
The CPU time is provided in seconds. To get the CPU time in us ( Like it is displayed on the E2 ui ) you need to multiply it by 1000000. Should I do it here already?
```suggestion
            e2Info.cpuTime = expression2.context.timebench * 1000000
```

Reply:
If CPU time is in seconds, then yeah this makes sense, might be a good call to round it to an int after though